core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -2604,6 +2672,10 @@ public Row newRow(long timestamp) {
         public Row newRow(long timestamp) {
             bumpMasterRef();
             if (timestamp < maxTimestamp) {
+                inOutOfOrderMode = true;
+                openTempPartition();
+                //TODO
+                // return tempOutOfOrderRow;
```

Comment:
Since we are throwing an exception, can we remove the commented code?

If not, maybe we can create an issue in GitHub and reference it so we can address it in the future? Like:

```java
// @TODO: return tempOutOfOrderRow rather than throwing an exception (#xxx)
```

Where `xxx` is the issue number.

Please ignore if it does not make sense.